L1Trigger/L1TGlobal/plugins/GenToInputProducer.cc

```diff
@@ -224,6 +234,7 @@ namespace l1t {
     //outputs
     std::unique_ptr<l1t::EGammaBxCollection> egammas(new l1t::EGammaBxCollection(0, bxFirst, bxLast));
     std::unique_ptr<l1t::MuonBxCollection> muons(new l1t::MuonBxCollection(0, bxFirst, bxLast));
+    std::unique_ptr<l1t::MuonShowerBxCollection> muonShowers(new l1t::MuonShowerBxCollection(0, bxFirst, bxLast));
```

Comment:
In the future, I think `std::make_unique<l1t::MuonShowerBxCollection>(...args...)` is preferred over `new`, but this is not the first/only example of that in this file.